internal/secrets/import.go

```diff
@@ -17,39 +17,38 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
-func ImportClusterIfEmpty(silent bool) (pkgtypes.Cluster, error) {
+func ImportClusterIfEmpty() (pkgtypes.Cluster, error) {
 	// find the secret in mgmt cluster's kubefirst namespace and read import payload and clustername
 	cluster := pkgtypes.Cluster{}
 	env, _ := env.GetEnv(constants.SilenceGetEnv)
 
-	var isClusterZero bool = true
```

Comment:
var isClusterZero bool = true
	if env.IsClusterZero == "false" {
		isClusterZero = false
	}. hey Patrick,this needs to be there as when we run this locally,we dont declare the isClusterzero as true explicitly in the env file,but when we run the api inside the cluster,we pass it as false

Reply:
Not sure I follow, if you see the lines below, it is asking `env.IsClusterZero` directly instead of having a hold-around kind of variable that serves no purpose other than the check that was there before.

I think what we can do is set it to `false` below for the check. This seems error prone though, so might give it a second pass.

Reply:
we want isClusterzero to be true by default as we dont declare it as true anywhere

Reply:
I was hoping that instead of telling me what code to write, to be told what is wrong. I see the previous code, do I understand this correctly?

* When this function executes, it checks imperatively if clusterzero is false
* If clusterzero is false, we "stop" this function from continuing otherwise it'll import certain stuff
* If clusterzero isn't false, we continue with the import process

Does that sum it up?

(Try to tell me what the behaviour is, not what code to write to get the behaviour)

Reply:
yes you are absolutely correct Patrick about the logic,but in second point you mentioned,if isclusterzero is false,we should continue and not stop

Reply:
Does the third bulletpoint change too? I can't imagine the program going:

* If clusterzero is false, we continue 
* If clusterzero is not false, we should also continue

Reply:
no if isclusterzero is not false we should return

Reply:
Fixed by changing the default value on the env variable declaration. The tl;dr is that we have a variable called `IS_CLUSTER_ZERO` that should always be defaulted to `true` but that wasn't specified in the code.

This commit https://github.com/konstructio/kubefirst-api/pull/391/commits/4338922e1690ece538f69751b3d88ba435bb04f4 changes that default value to true.